### PR TITLE
API for DNSEVER

### PR DESCRIPTION
**Pull request by flywithu**

### CODE REVIEW

File: dnsapi/dns_dnsever.sh [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  _info "Using dnsever remove"
+  _debug fulldomain "$fulldomain"
+  _debug txtvalue "$txtvalue"
+  if [ -z "$DNSEVER_ID" ] || [ -z "$DNSEVER_PW" ]; then
+    DNSEVER_ID=""
+    DNSEVER_PW=""
+    _err "You don't specify dnsever.com ID or PW yet."
+    _err "Please create you key and try again."
+    _err "export DNSEVER_ID=YOURID && export DNSEVER_PW=YOURPW  "
+    _err " ./acme.sh --issue --force --dns dns_dnsever -d "*.flywithu.com"  "
```
Comment (Neilpang) [inline dnsapi/dns_dnsever.sh:56]: remove this line

File: dnsapi/dns_dnsever.sh [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+  _debug fulldomain "$fulldomain"
+  _debug txtvalue "$txtvalue"
+
+  if [ -z "$DNSEVER_ID" ] || [ -z "$DNSEVER_PW" ]; then
+    DNSEVER_ID=""
+    DNSEVER_PW=""
+    _err "You don't specify dnsever.com ID or PW yet."
+    _err "Please create you key and try again."
+    _err "export DNSEVER_ID=YOURID && export DNSEVER_PW=YOURPW  "
+    _err " ./acme.sh --issue --force --dns dns_dnsever -d "*.flywithu.com"  "
```
Comment (Neilpang) [inline dnsapi/dns_dnsever.sh:30]: remove this line

File: dnsapi/dns_dnsever.sh [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+# Please Read this guide first: https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide
+
+#Usage: dns_myapi_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_dnsever_add() {
+  fulldomain=$1
+  txtvalue=$2
+  _info "Using dnsever add"
+  _debug fulldomain "$fulldomain"
+  _debug txtvalue "$txtvalue"
+
```
Comment (Neilpang) [inline dnsapi/dns_dnsever.sh:23]: https://github.com/acmesh-official/acme.sh/blob/master/dnsapi/dns_cf.sh#L21

please use `_readaccountconf_mutable `  and `_saveaccountconf_mutable `

**Comment by Neilpang** [comment]
read this guide first:
https://github.com/acmesh-official/acme.sh/wiki/DNS-API-Dev-Guide

File: dnsapi/dns_dnsever.sh (hunk 1 above)
Comment (flywithu) [reply dnsapi/dns_dnsever.sh]: fixed.

File: dnsapi/dns_dnsever.sh (hunk 3 above)
Comment (flywithu) [reply dnsapi/dns_dnsever.sh:23]: fixed.

**Comment by Neilpang** [comment]
can you please update to latest dev branch, and check the CI for the workflow DNS?
https://github.com/acmesh-official/acme.sh/wiki/DNS-API-Test

**Pull request closed without merging**